src/sections/Meshery/Meshery-integrations/Integration.style.js

```diff
@@ -1,93 +1,47 @@
 import styled from "styled-components";
 
 export const HoneycombGrid = styled.div`
-  display: flex;
-  --s: 9.375rem;
-  --m: 0.25rem;
-  --f: calc(var(--s) * 1.732 + 4 * var(--m) - 1px);
-
-  .container {
-    filter: drop-shadow(0px 4px 20px rgba(0, 0, 0, 0.1));
-    font-size: 0;
   
-  }
-
-  .container div {
+  .container-active{
+    display: flex;
+    height: 95%;
     background: white;
-    width: var(--s);
-    margin: var(--m);
-    height: calc(var(--s) * 1.1547);
-    display: inline-block;
-    font-size: initial;
-    clip-path: polygon(0% 25%, 0% 75%, 50% 100%, 100% 75%, 100% 25%, 50% 0%);
-    margin-bottom: calc(var(--m) - var(--s) * 0.2885);
+    justify-content: center;
+    align-items: center;
 
     :hover {
       transition: all 0.25s ease-in;
-      transform: scale(1.03);
+      transform: scale(1.03);    
       background: #00d3a9;
-      
-    
-    }
+        ::after{
+          content: "";
+          background: white;
+          margin: 0px 0px 0 6px;
+          clip-path: polygon(50% 0%,100% 25%,100% 75%,50% 100%,0% 75%,0% 25%);
+          height: 93%;
+          width: 93%;
```

Comment:
Finally, thank you!
The border width needs to be reduced a bit.
Probably by moderating the width, height, left, and top values a bit.

Reply:
Okay!